backend/presentation/filters.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import get_user_model
+from django_filters import rest_framework as filters
+
+from presentation.models import Presentation, Tag
+
+
+User = get_user_model()
+
+
+class PresentationFilter(filters.FilterSet):
+    scheduled_on = filters.IsoDateTimeFromToRangeFilter()
+    user = filters.ModelChoiceFilter(queryset=User.objects.filter(is_active=True))
+    tags = filters.ModelMultipleChoiceFilter(queryset=Tag.objects.all())
+
+    class Meta:
+        model = Presentation
+        fields = ["scheduled_on", "user", "tags"]
```

Comment:
It would probably be better to filter by the user's username instead of the user's id.
Same thing applies for tags.